### PR TITLE
Followup canceled jobs

### PR DESCRIPTION
**Pull request by IgorMazur**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,18 +1,10 @@
...
-	make force_release
+env:
+	@echo source .env/bin/activate
 
-force_release: clean
-	git push --tags
-	python setup.py sdist bdist_wheel
-	twine upload dist/*
+env_install:
+	pip install -r requirements.txt
```
Comment (IgorMazur) [inline Makefile]: Oh, I did not notice existing Makefile. Will revert it back.

**Comment by hmleal** [comment]
@IgorMazur the changes to the file are personal settings of your desktop. Set the global .gitignore your machine not the project gitignore ;)

```
.env/
*.sublime-project
*.sublime-workspace
```

**Comment by eoranged** [comment]
I don't really like the concept of cancelling one job using another one. Custom class for that purpose seems more reasonable solution.
Also there are still unrelated changes like environment-specific gitignores (might be handled by `.git/info/exclude` file or globally in user profile) and quite strange version bump.

**Comment by eoranged** [comment]
Closing this for the time being.
If you still want this to be merged please clean It up, rebase on master and provide additional arguments on why this should be implemented in rq_dashboard.

**Pull request closed without merging**